src/IFrameNavigator.ts

```diff
@@ -139,6 +146,7 @@ export interface UpLinkConfig {
   url?: URL;
   label?: string;
   ariaLabel?: string;
+  libraryIcon?: string;
```

Comment:
Maybe change this to "logo" since webpub-viewer isn't always going to be used by libraries?

Reply:
Also, should this be a URL instead?

Reply:
The `logoUrl` in `circulation-patron-web` is defined as a `string` but I can update web-pub viewer to expect a URL and just wrap the logoUrl in a new URL. https://github.com/NYPL-Simplified/circulation-patron-web/blob/7b73e4640f619616b2cfb5d649232ef0c7297b88/src/interfaces.ts#L50 

a URL will hopefully make it clear that SVG can't be passed in!

Reply:
I updated for the libraryIcon type to be URL instead of string.